src/js/isDiffArray-helper.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-env node */
+/*
+
+    Handlebars helper to determine whether a particular "diff" describes an array value or not.
+
+*/
+"use strict";
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.diff.helper.isDiffArray");
+
+gpii.diff.helper.isDiffArray.getHelperFunction = function () {
+    return function (diffDef, options) {
+        // TODO: Convert this to a single fluid.get check once https://issues.fluidproject.org/browse/FLUID-6217 is resolved.
```

Comment:
Release 3.0.0-dev.20171102T191539Z.c0636b395 should have this resolved

Reply:
Confirmed.  Thanks for the quick fix on that.